opa.yaml

```diff
@@ -0,0 +1,84 @@
+package:
+  name: opa
+  version: 0.64.1
+  epoch: 0
+  description: Open Policy Agent (OPA) is an open source, general-purpose policy engine.
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - go
+      - wolfi-baselayout
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/open-policy-agent/opa
+      tag: v${{package.version}}
+      expected-commit: 298f97d79da33ede2bd9ec9cbc5641daa63a8587
+
+  - uses: go/build
+    with:
+      ldflags: "-X github.com/open-policy-agent/opa/version.Version=${{package.version}}"
+      modroot: .
+      packages: .
+      output: opa
+
+  - uses: strip
+
+  - runs: |
+      mkdir -p ${{targets.contextdir}}/opa
+
+      cat << EOF > ${{targets.contextdir}}/opa/example.rego
+      package authz
+      import rego.v1
+
+      allow if {
+        input.path == ["users"]
+        input.method == "POST"
+      }
+
+      allow if {
+        input.path == ["users", input.user_id]
+        input.method == "GET"
+      }
+      EOF
+
+      cat << EOF > ${{targets.contextdir}}/opa/example_test.rego
+      package authz
+      import rego.v1
+
+      test_post_allowed if {
+        allow with input as {"path": ["users"], "method": "POST"}
+      }
+
+      test_get_anonymous_denied if {
+        not allow with input as {"path": ["users"], "method": "GET"}
+      }
+
+      test_get_user_allowed if {
+        allow with input as {"path": ["users", "bob"], "method": "GET", "user_id": "bob"}
+      }
+
+      test_get_another_user_denied if {
+        not allow with input as {"path": ["users", "bob"], "method": "GET", "user_id": "alice"}
+      }
+      EOF
```

Comment:
```suggestion
```

These two files can go in a directory called `opa` - e.g. `opa/example.rego` and `opa/example_test.rego`.

Then the test pipeline can invoke them directly.